packages/rif-id-core/test/util.ts

```diff
@@ -137,3 +153,17 @@ export const issueTestCredential = async (subject: string): Promise<Daf.Credenti
 
   return vc
 }
+
+export const importMnemonic = (agent: Daf.Agent, mnemonic: string) => ((agent.identityManager.getIdentityProviders()[0] as any)._kms.importMnemonic(mnemonic))
+
+export const startTestIssuerServer = (handle: any) => {
+  const app = express()
+  app.use(bodyParser.text())
+
+  app.post('/request_credential', function(req, res) {
+    const jwt = req.body
+    res.status(200).send('')
+  })
+
+  return app.listen(handle)
+}
```

Comment:
Maybe in the future we can mock `fetch` package instead of creating a new server

Reply:
Opened #51